README.md

```diff
@@ -56,6 +56,7 @@ To build and test the site locally, run:
 ```bash
 # install dependencies
 npm ci
+make module-init
```

Comment:
Readers should already have done that - see line 34.

We could add a comment to say something like ”make sure you have carried out the initial setup steps” and link back to Prerequisites.